.travis.yml

```diff
@@ -42,6 +43,7 @@ addons:
 
 env:
   global:
+    - DATABASE_URL=postgres://postgres:@localhost/cargo_registry_test
     - TEST_DATABASE_URL=postgres://postgres:@localhost/cargo_registry_test
```

Comment:
oh so if travis is using DATABASE_URL now, do we need to set TEST_DATABASE_URL anymore?